test/websocket-server.test.js

```diff
@@ -137,6 +137,25 @@ describe('WebSocketServer', function () {
     });
   });
 
+  describe('#address', function () {
+    it('can provide server address', function (done) {
+      const wss = new WebSocket.Server({ port: 0 }, () => {
+        const addr = wss.address();
+        assert.deepEqual(wss._server.address(), addr);
+        wss.close(done);
+      });
+    });
+
+    it(`will thrown exception if server was closed`, function (done) {
+      const wss = new WebSocket.Server({ port: 0, noServer: true }, () => {
+        assert.throws(() => {
+          wss.address();
+        }, /This server is operating in "noServer" mode/);
+        wss.close(done);
+      });
+    });
```

Comment:
Add a new test for the `address()` after `close()` case where `null` is returned.

Reply:
okidokie